templates/terraform/iam_policy.go.erb

```diff
@@ -210,6 +210,10 @@ func (u *<%= resource_name -%>IamUpdater) SetResourceIamPolicy(policy *cloudreso
 		return err
 	}
 
+<% if object.iam_policy.iam_policy_version -%>
+        json["version"] = <%= object.iam_policy.iam_policy_version -%>
```

Comment:
If we didn't need this here before, why do we now?

Reply:
This is for setIamPolicy(). The version is set outside of this file. Better to override it here for individual resource. Above two changes are for getIamPolicy().

Reply:
If it's being set outside of this file, then we should update the place where it _is_ set.

Reply:
good point!. let me check

Reply:
can you add a comment explaining that this is an override of the existing version that might have been set in the resource_iam_member|policy|binding code?